pom.xml

```
@@ -19,7 +26,42 @@
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
     <quarkus.platform.version>1.9.0.Final</quarkus.platform.version>
     <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+
+    <windup.web.scm.connection>scm:git:https://github.com/windup/windup-operator.git</windup.web.scm.connection>
```

I think `web` is not appropriate for the operator
```suggestion
    <windup.scm.connection>scm:git:https://github.com/windup/windup-operator.git</windup.web.scm.connection>
```

---

:+1: